### ISSUE TITLE
The envelopes endpoint doesn't handle a CTID with upper case letters. 

### ISSUE THREAD

**Issue by mparsons-ce** [CredentialEngine/CredentialRegistry#602]
We had a recent error where the finder import could not find an envelope. I checked and the resource had been published, but the CTID was in upper case. Of course every endpoint except the collection one arbitrarily sets the CTID to lower case before publishing. 

I checked found that the registry will resolve the resource (and graph) endpoints with an uppercase CTID:
https://sandbox.credentialengineregistry.org/resources/ce-17FCB5B2-7339-9B93-AA93-9741556E1481
but not the envelope
https://sandbox.credentialengineregistry.org/envelopes/ce-17FCB5B2-7339-9B93-AA93-9741556E1481

**Comment by excelsior** [comment]
@mparsons-ce Good catch! CTIDs are expected to be case-sensitive. I think all of the other endpoints abide to that rule. If you don't mind, I'd change the `GET /resources/:id` API to work the same way.

**Comment by mparsons-ce** [comment]
Our Api should set all CTIDs to lower case before publishing. I had missed doing the latter for collections.
We want the envelopes endpoint to be case insensitive like /graph and /resources.

**Comment by siuc-nate** [comment]
+1 to case-insensitivity with regard to CTIDs, even if they should always be lowercase.
